app/views/analyses/new/_genotype_data_file_help.html.haml

```diff
@@ -1,10 +1,11 @@
 .alert.alert-info
-  Genotype data needs to be provided in one of two accepted formats -
-  #{link_to 'VCF', 'http://samtools.github.io/hts-specs/', title: 'Variant Call Format', target: "_blank"} or
+  Genotype data needs to be provided in one of the accepted formats -
+  #{link_to 'VCF', 'http://samtools.github.io/hts-specs/', title: 'Variant Call Format', target: "_blank"},
+  #{link_to 'HapMap', 'http://augustogarcia.me/statgen-esalq/Hapmap-and-VCF-formats-and-its-integration-with-onemap/#hapmap', title: "HapMap", target: "_blank"} or
   #{link_to 'CSV', 'https://en.wikipedia.org/wiki/Comma-separated_values', title: 'Comma-separated values', target: "_blank"}.
   %br
   %br
-  VCF is a well-defined format and any such file should be acceptable. However, for best performance low quality
+  VCF and HapMap are well-defined formats and any such files should be acceptable. However, for best performance low quality
```

Comment:
Can you say "any such files"? "Any such file" maybe?

Reply:
Fixed.